scala-akka/src/test/scala/com/baeldung/scala/akka/scheduler/SchedulerSpec.scala

```diff
@@ -100,7 +100,7 @@ class SchedulerSpec
         system.scheduler.schedule(10.millis, 1.seconds, greeter, greet)
 
       val expectedMessage = Greeted("Lucifer MorningStar: Hello, Detective")
-      expectMsg(350.millis, expectedMessage)
+      expectMsg(600.millis, expectedMessage)
       //Cancel the schedule, should not get any more messages after that
```

Comment:
Did you intend for this change to be included in the PR?

Reply:
Sorry, accidental change. I reverted it